cairo/ethereum/cancun/vm/interpreter.cairo

```diff
@@ -320,3 +346,215 @@ func _execute_code{
     // Recursive call to continue execution
     return _execute_code(evm);
 }
+
+func process_message_call{
+    range_check_ptr,
+    bitwise_ptr: BitwiseBuiltin*,
+    keccak_ptr: KeccakBuiltin*,
+    poseidon_ptr: PoseidonBuiltin*,
+    env: Environment,
+}(message: Message) -> (MessageCallOutput, EthereumException*) {
+    alloc_locals;
+
+    // Check if this is a contract creation (target is empty)
+    if (cast(message.value.target.value.address, felt) == 0) {
+        let state = env.value.state;
+        let has_collision = account_has_code_or_nonce{state=state}(message.value.current_target);
```

Comment:
missing condition account_has_storage on `has_collision` condition

```
        is_collision = account_has_code_or_nonce(
            env.state, message.current_target
        ) or account_has_storage(env.state, message.current_target)
```

Reply:
oh good catch 
EELs are not up to date

Reply:
The update also included the usage of ethereum_rlp.
This broke a few RLP tests looking into it

Reply:
PR to update the EELS https://github.com/kkrt-labs/keth/pull/576
To merge before closing this thread